tests/python_tests/test_llm_pipeline.py

```diff
@@ -846,8 +846,9 @@ def test_pipelines_with_gguf_generate(pipeline_type, model_ids):
 
 @pytest.mark.parametrize("pipeline_type", get_gguf_pipeline_types())
 @pytest.mark.parametrize("model_ids", get_gguf_model_list())
+@pytest.mark.parametrize("enable_save_ov_model", get_gguf_enable_save_ov_model_property_list())
```

Comment:
```suggestion
@pytest.mark.parametrize("enable_save_ov_model", [True, False])
```
and that function can be just removed:)

Reply:
Done